delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/Main.scala

```diff
@@ -161,42 +164,7 @@ object Main extends BIOApp {
           )
         ) >> Task
           .traverse(plugins)(_.stop())
-          .timeout(30.seconds) >> KamonMonitoring.terminate >> terminateActorSystem()
-      }
-  }
-
-  private def terminateActorSystem()(implicit as: ActorSystemClassic): Task[Unit] =
-    Task.deferFuture(as.terminate()).timeout(15.seconds) >> Task.unit
-
-  implicit private def configReaderErrorHandler(failures: ConfigReaderFailures): UIO[ExitCode] = {
```

Comment:
We could open an issue to make sure that misconfiguration are displayed correctly ?